packages/internals/src/io.js

```diff
@@ -15,7 +13,7 @@ class IO {
 
 const io = fn => new IO(fn);
 
-module.exports = {
-  io,
-  IO
-};
+module.exports.io = io;
+module.exports.IO = IO;
+
+const { Lazy } = require("./lazy");
```

Comment:
Is there a reason the import is at the bottom of the file?

Reply:
This solves cyclic dependencies being undefined on import